packages/core/src/extensions/TableHandles/TableHandlesPlugin.ts

```diff
@@ -236,4 +458,86 @@ export class TableHandlesProsemirrorPlugin<
   public onUpdate(callback: (state: TableHandlesState) => void) {
     return this.on("update", callback);
   }
+
+  colDragStart = (event: {
```

Comment:
add comments that these functions are part of the public interface / what they're used for